### PR TITLE
Fix simple typos in feature-gates.md (#18275)

### PR DESCRIPTION
**Pull request by hase1128**
I fixed few typos in feature-gates.md
Related issue is https://github.com/kubernetes/website/issues/18275

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -374,7 +374,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `DynamicAuditing`: Enable [dynamic auditing](/docs/tasks/debug-application-cluster/audit/#dynamic-backend)
 - `DynamicKubeletConfig`: Enable the dynamic configuration of kubelet. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/).
 - `DynamicProvisioningScheduling`: Extend the default scheduler to be aware of volume topology and handle PV provisioning.
-  This feature is superceded by the `VolumeScheduling` feature completely in v1.12.
+  This feature is superseded by the `VolumeScheduling` feature completely in v1.12.
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Just for people who are curious as I am: https://grammarist.com/spelling/supersede-or-supercede/

**Comment by gochist** [comment]
Re-assign to the next [PR wrangler after winter holidays](https://github.com/kubernetes/website/wiki/PR-Wranglers)

/unassign
/assign @mistyhacks

**Comment by hase1128** [comment]
@sftim 
Could you add `approve` label to this issue?

**Comment by sftim** [comment]
As of yesterday (#18398) yes I can.
/approve

**Comment by hase1128** [comment]
> As of yesterday (#18398) yes I can.
> /approve

It's a great news!!

**Pull request merged**